### ISSUE TITLE
Getting Started with ACH

### ISSUE THREAD

**Issue by shaigluskin** [iATSPayments/com.iatspayments.civicrm#64]
Hi Folks,

I'm trying to get started with ACH. My clients IATS account is set up for ACH use.

I thought what I had to do was add a new payment processor to Civi. But I can't seem to do that. I clicked "Add new processor" and then chose "iATS Payments ACH/EFT" and I used the same uid and password for IATS and the URLs were prefilled. When I saved it, it overwrote my existing iats credit card payment processor. Then I tried editing that to return it to the cardit card settings because it is a live site and the credit card payment processor is used extensively. After changing back the settings for the credit card, the field "Processor type" keeps reverting back to iATS Payments ACH/EFT; I can't get credit card to stick in that dropdown. But the credit card processing is still working even though the processor type is wrong.

In short, how do I get starrted with adding ACH processing to my client's site.

Thanks so much.

Shai

**Comment by adixon** [comment]
You're doing the right thing, there's something wanky with your civicrm.

I have found that if I'm creating a new entity and then open a different window with an existing one, it'll sometimes use that entity's id.

Try writing down your credentials rather than opening that other window and copying, perhaps ...

You can also manually just look at the civicrm_payment_processor table to see what's up.
